attributes/default.rb

```diff
@@ -35,6 +35,7 @@
 default['cluster']['launch_templates_config_path'] = "#{node['cluster']['shared_dir']}/launch-templates-config.json"
 default['cluster']['instance_types_data_path'] = "#{node['cluster']['shared_dir']}/instance-types-data.json"
 default['cluster']['computefleet_status_path'] = "#{node['cluster']['shared_dir']}/computefleet-status.json"
+default['cluster']['fleet_config_path'] = "#{node['cluster']['shared_dir']}/fleet-config.json"
```

Comment:
For now I'd store this file to a local directory such as /etc/parallelcluster/slurm_plugin`

Reply:
Moved into `/etc/parallelcluster` with this patch: https://github.com/aws/aws-parallelcluster-cookbook/pull/1508
I didn't move it into `slurm_plugin` because in future this file would be used by other schedulers too.